### PR TITLE
[offline-upgrade] Fix strings for l10n

### PR DESCRIPTION
**Pull request by cam-rod**
Correcting changes made to some strings in #467 for localization.

### CODE REVIEW

**Comment by piotrdrag** [comment]
Thanks!

**Comment by jan-kolarik** [comment]
Just maybe assign the message itself into some variable, like `message = _("Upgrade complete! Cleaning up and powering off...")` and then use it both in the `Plymouth` and `log_status` calls so we are not repeating it twice...

**Comment by cam-rod** [comment]
Sorry about the delay, made the suggested changes.

File: plugins/system_upgrade.py [hunk 1]
```diff
@@ -701,11 +701,13 @@ def transaction_download(self):
...
+        if self.state.poweroff_after:
+            upgrade_complete_msg = _("Upgrade complete! Cleaning up and powering off...")
+        else:
+            upgrade_complete_msg = _("Upgrade complete! Cleaning up and rebooting...")
 
-        Plymouth.message(_("Upgrade complete! Cleaning up and " + power_op + "..."))
-        self.log_status(_("Upgrade complete! Cleaning up and " + power_op + "..."),
-                        UPGRADE_FINISHED_ID)
+        Plymouth.message(upgrade_complete_msg)
+        self.log_status(upgrade_complete_msg)
```
Comment (jan-kolarik) [inline plugins/system_upgrade.py:710]: You've accidentally dropped the `UPGRADE_FINISHED_ID` in the `log_status` call :slightly_smiling_face:

Comment (cam-rod) [reply plugins/system_upgrade.py]: whoops, fixed

**Comment by jan-kolarik** [comment]
Thanks, LGTM!

**Pull request merged**